swarm/pss/notify/notify_test.go

```diff
@@ -51,137 +52,136 @@ func TestStart(t *testing.T) {
 		ID:             "0",
 		DefaultService: "bzz",
 	})
-	l_nodeconf := adapters.RandomNodeConfig()
-	l_nodeconf.Services = []string{"bzz", "pss"}
-	l_node, err := net.NewNodeWithConfig(l_nodeconf)
+	leftNodeConf := adapters.RandomNodeConfig()
+	leftNodeConf.Services = []string{"bzz", "pss"}
+	leftNode, err := net.NewNodeWithConfig(leftNodeConf)
 	if err != nil {
 		t.Fatal(err)
 	}
-	err = net.Start(l_node.ID())
+	err = net.Start(leftNode.ID())
 	if err != nil {
 		t.Fatal(err)
 	}
 
-	r_nodeconf := adapters.RandomNodeConfig()
-	r_nodeconf.Services = []string{"bzz", "pss"}
-	r_node, err := net.NewNodeWithConfig(r_nodeconf)
+	rightNodeConf := adapters.RandomNodeConfig()
+	rightNodeConf.Services = []string{"bzz", "pss"}
+	rightNode, err := net.NewNodeWithConfig(rightNodeConf)
 	if err != nil {
 		t.Fatal(err)
 	}
-	err = net.Start(r_node.ID())
+	err = net.Start(rightNode.ID())
 	if err != nil {
 		t.Fatal(err)
 	}
 
-	err = net.Connect(r_node.ID(), l_node.ID())
+	err = net.Connect(rightNode.ID(), leftNode.ID())
 	if err != nil {
 		t.Fatal(err)
 	}
 
-	l_rpc, err := l_node.Client()
+	leftRpc, err := leftNode.Client()
 	if err != nil {
 		t.Fatal(err)
 	}
 
-	r_rpc, err := r_node.Client()
+	rightRpc, err := rightNode.Client()
 	if err != nil {
 		t.Fatal(err)
 	}
 
-	var l_addr string
-	err = l_rpc.Call(&l_addr, "pss_baseAddr")
+	var leftAddr string
+	err = leftRpc.Call(&leftAddr, "pss_baseAddr")
 	if err != nil {
 		t.Fatal(err)
 	}
 
-	var r_addr string
-	err = r_rpc.Call(&r_addr, "pss_baseAddr")
+	var rightAddr string
+	err = rightRpc.Call(&rightAddr, "pss_baseAddr")
 	if err != nil {
 		t.Fatal(err)
 	}
 
-	var l_pub string
-	err = l_rpc.Call(&l_pub, "pss_getPublicKey")
+	var leftPub string
+	err = leftRpc.Call(&leftPub, "pss_getPublicKey")
 	if err != nil {
 		t.Fatal(err)
 	}
 
-	err = r_rpc.Call(nil, "pss_setPeerPublicKey", l_pub, controlTopic, l_addr)
+	var rightPub string
+	err = rightRpc.Call(&rightPub, "pss_getPublicKey")
 	if err != nil {
 		t.Fatal(err)
 	}
 
 	rsrcName := "foo.eth"
 	rsrcTopic := pss.BytesToTopic([]byte(rsrcName))
 
+	time.Sleep(time.Second)
```

Comment:
suspicious, comment?

Reply:
just the usual "healthy" bullshit

Reply:
Well, I don't mean "healthy" exactly, I mean wait for the bzz handshakes to complete and the peer entry to be put in the kademlia table. Without sleep I don't think it can be guaranteed?